Source/SwiftLintFramework/Rules/EmptyParenthesesWithTrailingClosureRule.swift

```diff
@@ -24,7 +24,12 @@ public struct EmptyParenthesesWithTrailingClosureRule: ASTRule, ConfigurationPro
             "[1, 2].map({ $0 + 1 })\n",
             "[1, 2].reduce(0) { $0 + $1 }",
             "[1, 2].map { number in\n number + 1 \n}\n",
-            "let isEmpty = [1, 2].isEmpty()\n"
+            "let isEmpty = [1, 2].isEmpty()\n",
+            "UIView.animateWithDuration(0.3, animations: {\n" +
+            "   self.disableInteractionRightView.alpha = 0\n" +
+            "}, completion: { _ in\n" +
+            "   ()\n" +
```

Comment:
that's interesting! We didn't mean any offense by that, in fact it might be better to do than what I usually do in these cases, which is to add a `/* no-op */` comment.